content/docs/command-reference/plots/index.md

```diff
@@ -147,9 +150,9 @@ header (first row) are equivalent to field names.
   `index` field (explained above) is the default.
 
 - `<DVC_METRIC_Y>` (optional) - field name of the data for the Y axis. It can be
-  defined with the `-y` option of the `dvc plot` subcommands. The default is the
-  last one found in the metrics file: the last column for CSV/TSV, or the last
-  field for JSON/YAML.
+  defined with the `-y` option of the `dvc plot` subcommands. It defaults to the
+  last one found in the first row of the metrics file: the last column for
+  CSV/TSV, or the last field for JSON/YAML.
```

Comment:
> It defaults to the last one found in the first row of the metrics file: the last column for CSV/TSV

Questions:

- What about CSV/TSV files that have a headers row though? Is it the last header from that first row, or some un-named column in case the first data row is longer than headers?
- What about CSV/TSV files without a headers row? See `dvc plots show --no-header`.

Reply:
In any case,

> last one found in the first row of the metrics file

is too hard to read. Let's find a way to simplify that is correct even if it doesn't have all the implementation details, e.g. `last header found in the metrics file`.

Reply:
> What about CSV/TSV files that have a headers row though? Is it the last header from that first row, or some un-named column in case the first data row is longer than headers?

It will be last header from first row.

> What about CSV/TSV files without a headers row? See dvc plots show --no-header

Again it will be last value from first row. Suppose first row has 6 values. So in each row every 6th value will be used to generate the plot.

Reply:
Sounds good, resolving this one.